go.mod

```diff
@@ -31,7 +31,7 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
 // At least until GetABCIEventHistory() is implemented and released.
 // And also `gentx --keyring-dir=...`
-replace github.com/cosmos/cosmos-sdk => github.com/agoric-labs/cosmos-sdk v0.43.0-rc0.agoric
+replace github.com/cosmos/cosmos-sdk => github.com/agoric-labs/cosmos-sdk v0.43.0-rc2.agoric
```

Comment:
why are we upgrading? I see

> Take most recent cosmos-sdk with our changes

Which changes?

Where can I see the difference between rc0 and rc2? I don't see rc2 in https://github.com/agoric-labs/cosmos-sdk/releases

Reply:
> Which changes?

https://github.com/agoric-labs/cosmos-sdk/tags

It's not a published release, just a tag.

Reply:
thans... found https://github.com/cosmos/cosmos-sdk/compare/v0.43.0-rc0...agoric-labs:v0.43.0-rc2.agoric ... not yet enlightened...